Testscripts/Windows/VERIFY-INFINIBAND-MultiVM.ps1

```diff
@@ -165,6 +172,16 @@ function Main {
 			return "SKIPPED"
 		}
 
+		if ($installOFEDFromExtension) {
+			foreach ($VMData in $AllVMData) {
+				$install_Output = Set-AzVMExtension -ResourceGroupName $VMData.ResourceGroupName -Location $TestLocation -VMName $VMData.RoleName `
+					-ExtensionName "InfiniBandDriverLinux" -Publisher "Microsoft.HpcCompute" -Type "InfiniBandDriverLinux" -TypeHandlerVersion "1.0"
```

Comment:
Check the latest TypeHandler version. It's 1.1 or 1.2. Don't know if there's a way to automatically use the latest one.